.circleci/config.yml

```diff
@@ -1763,6 +1763,19 @@ workflows:
           parallelism: 6
           xdist: 1
 
+      # report api tests
+      - tox-base:
+          requires:
+            - "unit-tests"
+          matrix:
+            parameters:
+              python_version_major: [3]
+              python_version_minor: [8, 9, 10, 11, 12]
```

Comment:
why do you need to test it on all the python versions?

Reply:
We can do just 3.8 if it's too expensive